oauth2client/contrib/gce.py

```diff
@@ -135,6 +136,8 @@ def _refresh(self, http_request):
                 raise HttpAccessTokenRefreshError(str(e),
                                                   status=response.status)
             self.access_token = token_content['access_token']
+            delta = datetime.timedelta(seconds=int(token_content['expires_in']))
```

Comment:
Historical note: it seems we use `expires_in` in this library consistently, so no need to rock the boat.